
uint8 STATUS_PAUSED = 0
uint8 STATUS_RUNNING = 1


Header header

uint8 status
duration duration
duration current
time currentTime
time startTime
time endTime
string[] bagfiles

PlayTopicStatus[] topics

